### PR TITLE
Review v2

### PR DESCRIPTION
**Pull request by mrjuda**
Retky's FCC Product Landing Page (Microverse's trials project) was copied for review purposes only.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+  <body>
+   <div id="header">
+     <img id="header-img" src="https://helpx.adobe.com/content/dam/help/en/photoshop/using/convert-color-image-black-white/jcr_content/main-pars/before_and_after/image-before/Landscape-Color.jpg" alt="">
+     <nav id="nav-bar">
+         <a class="nav-link" href="#form">Subscribe</a>
+         <a class="nav-link" href="#video">How It Works</a>
+         <a class="nav-link" href="#pricing">Price</a>
+     </nav>
+   </div>
```
Comment (mrjuda) [inline index.html:17]: I think it would be nicer if you use the 'header' tag instead of a regular 'div'.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```
Comment (mrjuda) [inline index.html:57]: In order to keep your code clean, it would be nicer to remove the comments

**Review by mrjuda — COMMENTED** [review]
You're doing a great job. However, there are a few things that could be improved. Please, kindly check out the comments on this review.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+  border: solid;
+  height: 200px;
+}
+#form{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
```
Comment (mrjuda) [inline css/style.css:55]: According to [CSS & HTML best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly avoid the overuse of IDs. Instead, try to use classes.

**Review by mahbubul14 — APPROVED** [review]
# Project Approved :trophy: :tada: 

Hi @mrjuda  , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :100: 
Congratulations! 🎉 

:heavy_check_mark: Used correct GitHub flow.
:heavy_check_mark: Communicated in a professional way.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**